pkg/operator/controller/externaldns/deployment_test.go

```diff
@@ -2290,6 +2313,7 @@ func TestEnsureExternalDNSDeployment(t *testing.T) {
 		name               string
 		existingObjects    []runtime.Object
 		expectedExist      bool
+		ExpectedSecret     corev1.Secret
```

Comment:
This field is not used in the test case, should be removed.

Reply:
Done here 6f4d092af25d1c2fc93cb34347d5e03f794cf83d